features/user/index.ts

```diff
@@ -0,0 +1,6 @@
+export * from './components'
+export * from './repository'
+export * from './service'
+export * from './actions'
```

Comment:
Wykorzystujesz to wszystko poza featurem user? Z feature exportujesz tylko to co używasz poza tym featurem, czyli w twoim przypadku chyba tylko components ale nie jestem pewny